include/mbgl/util/logging.hpp

```diff
@@ -36,6 +37,11 @@ class Log {
     }
 
 public:
+    Log();
+    ~Log() = default;
```

Comment:
destructor of a complex class shall be put to .cpp

Reply:
Done